### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by jorgegoco**
- Added medium-fidelity wireframe in HTML file.
- Created the JS file for the dynamic workaround of the HTML file with the use of classes.
- Used localStorage to preserve the book's data in the browser.
- Fixed linter errors.

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
+    this.arr.push({ title, author });
+  }
 
-function addBook(title, author) {
-  const newBook = new Book(title, author);
-  books.push(newBook);
-}
+  removeBook(title, author) {
+    this.arr = this.arr.filter((elem) => elem.title !== title && elem.author !== author);
+  }
```
Comment (jerryowusu) [inline assets/scripts/main.js]: Removing a book also remove all other books with the same title and/or author. :-1:  Kindly consider adding a unique `id` to each element to fix this issue and make sure only the selected book is removed. :pray:

File: assets/scripts/main.js [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
+    this.arr = JSON.parse(localStorage.getItem('booksData')) || [];
+  }
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
+  addBook(title, author) {
+    this.arr.push({ title, author });
+  }
```
Comment (jerryowusu) [inline assets/scripts/main.js:8]: [OPTIONAL] The array methods like push, pop,  splice, etc can mutate/or change the array. Kindly consider using unmutabel methods like concat, filter, and slice.

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @jorgegoco 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

almost-paul-walker


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Well organised code
:heavy_check_mark: Followed correct GitHub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 Hi there  @jorgegoco!
 
You have successfully completed this project! There is nothing else to say other than... it's time to merge it! rocket Time to celebrate, you have done a fantastic job, congratulations! :tada:

  
 ##  Highlights 
 * Great work using the right GitHub Flow :100:
 * Awesome file organization :card_file_box:
 * Commits are descriptive, good work.
 * Linters are working properly, great job!

 APPROVED! :100: :100: :100:

## Optional suggestions 
 _Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 100_
 
 * [ ]  **[OPTIONAL]:
 -N/A

Cheers and best of luck with the following projects! :raised_hands:
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me **@alfredbis29** in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**